src/scripts.js

```diff
@@ -59,14 +68,20 @@ const updateUser = (currentUser, users) => {
 }
 
 const displayStats = (currentUser) => {
-  console.log(currentUser.hydration.days)
-  statsSection.innerHTML = `
-  <article id="todaysIntake">
-    <p>Your water intake for today is: ${currentUser.hydration.getDaily(currentUser.hydration.days[currentUser.hydration.days.length - 1].date)}
-  </article>`
-let weeklyIntake = document.createElement('article')
-currentUser.hydration.getWeekly().forEach(day => weeklyIntake.innerHTML += `<p>On ${day.date} you drank: ${day.numOunces} fl oz</p>`)
-  statsSection.appendChild(weeklyIntake);
+  todaysIntake.innerHTML +=
+  `<p>Your water intake for today is: ${currentUser.hydration.getDaily(currentUser.hydration.days[currentUser.hydration.days.length - 1].date)}`
+  currentUser.hydration.getWeekly().forEach(day => weeklyIntake.innerHTML += `<p>On ${day.date} you drank: ${day.numOunces} fl oz</p>`);
+  avgSleepHours.innerHTML +=
+  `<p>Your total sleep hour average is: ${currentUser.sleep.getAverage()}`
+  avgSleepQuality.innerHTML +=
+  `<p>Your total sleep quality average is: ${currentUser.sleep.getAverageQuality()}`
+  todaysSleepHours.innerHTML +=
+  `<p>You slept ${currentUser.sleep.getSleep(currentUser.sleep.days[currentUser.sleep.days.length - 1].date)} hours last night.`
+  todaysSleepQuality.innerHTML +=
+  `<p>Your quality last night was: ${currentUser.sleep.getSleepQuality(currentUser.sleep.days[currentUser.sleep.days.length - 1].date)}.`
+  currentUser.sleep.getWeekSleep(currentUser.sleep.days[currentUser.sleep.days.length - 8].date).forEach(day => weeklySleepHours.innerHTML += `<p>On ${day.date}, you slept ${day.hoursSlept} hours.`);
+  currentUser.sleep.getWeekQuality(currentUser.sleep.days[currentUser.sleep.days.length - 8].date).forEach(day => weeklySleepQuality.innerHTML += `<p>On ${day.date}, your sleep quality was: ${day.sleepQuality}.`);
+
```

Comment:
I think we could also integrate these <p> tags into our HTML structure, and just wrap the interpolated(calculated) value into a span, which has the ID which we use for the selector. Could clean up the length of lines of the JS files.